beta-src/src/components/ui/WDInfoPanel.tsx

```diff
@@ -35,7 +45,18 @@ const WDInfoPanel: React.FC<WDInfoPanelProps> = function ({
       [voteKey]: !voteState[voteKey],
     };
 
+    const currentGameID = gameID.toString();
+    const countryID = userCountry.countryID.toString();
```

Comment:
Same wrap it in `String()` since were just casting it to a different type.

Reply:
Thank you so much for this!!! Did not know this!